corehq/apps/receiverwrapper/tests/test_submissions.py

```diff
@@ -13,14 +13,6 @@
 from corehq.form_processor.tests.utils import run_with_all_backends
 
 
-# bit of a hack, but the tests optimize around this flag to run faster
-# so when we actually want to test this functionality we need to set
-# the flag to False explicitly
-from corehq.toggles import USE_SQL_BACKEND, NAMESPACE_DOMAIN
-from toggle.shortcuts import update_toggle_cache, clear_toggle_cache
-
-
-@override_settings(UNIT_TESTING=False)
```

Comment:
it seems like different code gets run when this is set to `False` (based on the comment)? having the tests pass might be a misnomer. will let @czue respond